### DISCUSSION TITLE
how to convert excelize to bytes.

### DISCUSSION THREAD

**Discussion by WableSanket** [qax-os/excelize#1348] [General]
How to convert excelize to []byte ?

**Comment by xuri** [comment]
Did you mean that get the generated workbook content as `[]byte` data type, if that, you can use the [`Write`](https://pkg.go.dev/github.com/xuri/excelize/v2@master#File.Write) function like this:

```go
buf := new(bytes.Buffer)
if err := f.Write(buf); err != nil {
    // handle the error here
}
content := buf.Bytes() // []byte
```

**Reply by WableSanket** [reply]
Thanks
